pom.xml

```diff
@@ -22,323 +22,34 @@
-                    <artifactId>maven-war-plugin</artifactId>
-                    <version>${version.plugin.maven-war-plugin}</version>
-                </plugin>
-
-                <!-- Felix plugins -->
-
-                <plugin>
-                    <groupId>org.apache.felix</groupId>
-                    <artifactId>maven-bundle-plugin</artifactId>
-                    <version>${version.plugin.felix.maven-bundle-plugin}</version>
-                    <extensions>true</extensions>
-                </plugin>
-
-                <!-- Karaf plugins -->
-
-                <plugin>
-                    <groupId>org.apache.karaf.tooling</groupId>
-                    <artifactId>karaf-maven-plugin</artifactId>
-                    <version>${version.org.apache.karaf}</version>
-                </plugin>
-
-                <!-- MojoHaus plugins -->
-
-                <plugin>
-                    <groupId>org.codehaus.mojo</groupId>
-                    <artifactId>build-helper-maven-plugin</artifactId>
-                    <version>${version.plugin.mojo.build-helper-maven-plugin}</version>
-                </plugin>
-
-                <!-- Other plugins -->
-
-                <plugin>
-                    <groupId>org.asciidoctor</groupId>
-                    <artifactId>asciidoctor-maven-plugin</artifactId>
-                    <version>${version.plugin.asciidoctor-maven-plugin}</version>
-                </plugin>
-
-            </plugins>
-        </pluginManagement>
-
-        <plugins>
-            <!-- Core plugins -->
-
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-compiler-plugin</artifactId>
-                <configuration>
-                    <source>1.8</source>
-                    <target>1.8</target>
-                </configuration>
-            </plugin>
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-surefire-plugin</artifactId>
-                <configuration>
-                    <!-- Skip tests by default and enable them only in "qtest" profile -->
-                    <skipTests>${skip.quickstarts.tests}</skipTests>
-                </configuration>
-            </plugin>
-
-            <!-- Felix plugins -->
-
-            <plugin>
-                <groupId>org.apache.felix</groupId>
-                <artifactId>maven-bundle-plugin</artifactId>
-                <version>${version.plugin.felix.maven-bundle-plugin}</version>
-                <inherited>true</inherited>
-                <extensions>true</extensions>
-                <configuration>
-                    <obrRepository>NONE</obrRepository>
-                    <instructions>
-                        <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
-                    </instructions>
-                </configuration>
-            </plugin>
-        </plugins>
-    </build>
-
     <modules>
```

Comment:
Why explicitly listing the sub sub module?

Reply:
My thought here is this 

- we don't want the quickstarts to have parents - if you remove parentage from the quickstarts, then developers can grab them quickly and adapt them to whatever scenario they want
- we do want to build the quickstarts all in one bunch

Reply:
We can build each of them except custom-distro